mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSModelAsNode.kt

```diff
@@ -114,7 +114,7 @@ data class MPSModelAsNode(val model: SModel) : IDefaultNodeAdapter {
         } else if (property.isIdProperty()) {
             reference.serialize()
         } else if (property.conformsTo(BuiltinLanguages.MPSRepositoryConcepts.Model.stereotype)) {
-            model.name.stereotype
+            model.name.stereotype.ifEmpty { null }
```

Comment:
I'm not sure why you are doing this, but I had an issue related to stereotype (and other properties) in #511 (specifically 709c30d1623d09cf5703775c7c25d620d1b3ee7e ) because the properties were initially not properly listed.